com.unity.render-pipelines.universal/Runtime/UniversalRenderPipelineCore.cs

```diff
@@ -160,24 +290,70 @@ public bool IsCameraProjectionMatrixFlipped()
             return true;
         }
 
+        /// <summary>
+        /// The sorting criteria used when drawing opaque objects by the internal URP render passes.
+        /// When a GPU supports hidden surface removal, URP will rely on that information to avoid sorting opaque objects front to back and
+        /// benefit for more optimal static batching.
```

Comment:
This could probably benefit from a "seealso: sorting criteria" as it is a complicated topic.

Reply:
fixed